### PR TITLE
feat(functions/v2): update imagemagick sample

### PR DESCRIPTION
**Pull request by saraford**
I was having trouble with the tests, so @ace-n and I agreed that he'd make sure the tests pass.

### CODE REVIEW

**Review by ace-n — APPROVED** [review]
LGTM, but I contributed some of the code - so I'd like another set of eyes on this.

@averikitsch PTAL? 🙂

File: functions/imagemagick/index.js [hunk 1]
```diff
@@ -29,12 +30,12 @@ const {BLURRED_BUCKET_NAME} = process.env;
...
 // [START functions_imagemagick_analyze]
 // Blurs uploaded images that are flagged as Adult or Violence.
-exports.blurOffensiveImages = async event => {
+functions.cloudEvent('blurOffensiveImages', async cloudEvent => {
   // This event represents the triggering Cloud Storage object.
-  const object = event;
-
-  const file = storage.bucket(object.bucket).file(object.name);
-  const filePath = `gs://${object.bucket}/${object.name}`;
+  const bucket = cloudEvent.data.bucket;
```
Comment (averikitsch) [inline functions/imagemagick/index.js:35]: nit, we may want to check that data, bucket and name exist

**Review by averikitsch — APPROVED** [review]

File: functions/imagemagick/index.js (hunk 1 above)
Comment (ace-n) [reply functions/imagemagick/index.js:35]: I'd argue this is distracting to users **if** we presume events are **a)** created programmatically and **b)** only come from `GCS -> Eventarc` to this function.

(In other words, we shouldn't have any _event-structure-related human errors_ outside of testing scenarios.)

Comment (ace-n) [reply functions/imagemagick/index.js:35]: Resolved offline.

**Review by bcoe — APPROVED** [review]

**Comment by ace-n** [comment]
Failures are unrelated; admin-merging.

**Pull request merged**